test/queue_IO_check.cxx

```diff
@@ -242,6 +246,14 @@ main(int argc, char* argv[])
     return 1;
   }
 
+  if (vm.count("nelements")) {
+    nelements = vm["nelements"].as<int>();
+
+    if (nelements <= 0) {
+      throw std::domain_error("# of elements must be a positive integer");
```

Comment:
throw std::... is discouraged; we might want to concoct and throw an ERS Issue here

Reply:
Replaced the throws of std::domain_error with 
```
ERS_DECLARE_ISSUE(appfwk,                 ///< Namespace                                                                  
                  ParameterDomainIssue,  ///< Issue class name                                                            
                  "ParameterDomainIssue: \"" << ers_messg << "\"",
                  ((std::string)ers_messg))
```